### PR TITLE
[react-native-snap-carousel] Update to 3.6

### PR DESCRIPTION
**Pull request by gazaret**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/archriss/react-native-snap-carousel/tree/v3.6.0>>
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/react-native-snap-carousel/index.d.ts [hunk 1]
```diff
@@ -164,22 +159,24 @@ export interface CarouselProps<T> extends React.Props<ScrollViewProperties> {
...
-    animationFunc?: 'decay' | 'timing' | 'spring';
+    activeAnimationOptions?: Animated.DecayAnimationConfig | Animated.TimingAnimationConfig | Animated.SpringAnimationConfig;
     /**
-     * Animation options to be merged with the default ones. Can be used w/ animationFunc
+     * Custom animation type: either 'decay, 'spring' or 'timing'.
+     * Note that it will only be applied to the scale animation since opacity's animation type will always be set
+     * to timing (no one wants the opacity to 'bounce' around)
      */
-    customAnimationOptions?: Animated.DecayAnimationConfig | Animated.TimingAnimationConfig | Animated.SpringAnimationConfig;
+    activeAnimationType?: string;
```
Comment (jnbt) [inline types/react-native-snap-carousel/index.d.ts]: Should we use `activeAnimationType?: 'decay' | 'spring' | 'timing'`?

**Review by jnbt — COMMENTED** [review]
@gazaret LGTM. My only idea would be to use a fixed set of valued for `activeAnimationType`. WDYT?

**Comment by jnbt** [comment]
@gazaret Thanks!

**Review by jnbt — APPROVED** [review]

**Comment by gazaret** [comment]
@jnbt thanks for the review!

**Pull request merged**